main/test/flix/Test.Exp.ExtMatch.flix

```diff
@@ -0,0 +1,19 @@
+mod Test.Exp.ExtMatch {
+
+    @test
+    def testExtMatch01Int32(): Bool = {
+        ematch (xvar A(1)) {
+            case A(x) => x == 1
+            case A(_) => false
+        }
+    }
```

Comment:
@magnus-madsen do you mean this?

Reply:
I need it nested with multiple matches. And for the tests to pass.

Reply:
Rights. Btw it's TypeVerifier that fails right now with `???`

Reply:
Can it be fixed?

Reply:
Yes will do that shortly and we can get this merged in